Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.yml

```
@@ -0,0 +1,60 @@
+category: Authentication
+commonfields:
+  id: ThycoticDSV
+  version: -1
+configuration:
+- display: Server URL to DSV (e.g. https://example.com)
```

Does the server URL usually have a specific structure?
```suggestion
- display: Server URL (e.g. https://example.com)
```

---

no, it does not